src/main/java/chainOfResponsability/VerifyDate.java

```diff
@@ -0,0 +1,17 @@
+package chainOfResponsability;
+
+public class VerifyDate extends HandlerData {
+    private String date;
+
+    public VerifyDate(String date) {
+        this.date = date;
+    }
+
+    public ShowData dataValidation(DataEntry data) {
```

Comment:
here we should have `DataEntry data` and `DataOutput output` where we have the appropriate value types and the responsibility of each of this sanitizers is to populate the data as needed.